portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,62 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Holly Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="introduction">
+        <h1>Hi there! </h1>
+
+        <div id="headshotWrapper">
+            <img id="headshot" src="city.jpg" alt="city skyline"/> 
+            <div id="caption"> My name is Holly, and this is the small place on the Internet that has my portfolio. </div> 
```

Comment:
nit: Unnecessary space before My, and after the period.

Reply:
Thank you for pointing this out! I was wondering if the extra space is a consideration from the coding style perspective? Or should I think about avoiding extra spaces in terms from the perspective of the page's layout, ie. extra spaces will cause some layouts to not look good on the screen?

Reply:
I found a relevant section from the Google style guide
https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace

Reply:
Thank you for pointing me to the style guides. I just pushed a new commit with updates, please see the commit message for more detailed changes.